contrib/html_demo/JupyterCode/13_image_similarity_export.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li>This needs to first create the ../visualize/data directory, since throws error otherwise.</li><li>[minor] visualize does not seem a very informative name. Maybe call it "similarity_data"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Completed